src/authors/authors.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+    <h1>Lists of Authors</h1>
+    <div class="wrap">
```

Comment:
I would suggest moving it before the `<h1>` and changing it to `main` and naming the class `main`.